.github/workflows/semgrep.yml

```diff
@@ -0,0 +1,18 @@
+name: Semgrep
+
+on:
+  pull_request: {}
+  push:
+    branches: ["master"]
+
+jobs:
+  semgrep:
+    name: Semgrep
+    runs-on: ubuntu-latest
```

Comment:
Is it possible to run on `debian-latest`?

Reply:
Github-hosted runners do not include debian-latest.